rero_ils/modules/documents/templates/rero_ils/detailed_view_documents.html

```diff
@@ -34,7 +34,7 @@ <h1 class="col-sm-10 col-md-11">{{ record.title }}</h1>
             <dl class="row mb-0">
             <!-- AUTHORS -->
             {% if record.authors|length > 0 %}
-              {{ dl(_('Author'), record.authors|authors_format) }}
+              {{ dl(_('Author'), record.pid|authors_format) }}
```

Comment:
an other approach can simply: `record.replace_refs()|authors_format`. This avoid to modify the filter.